product-service/handlers/createProduct.js

```diff
@@ -0,0 +1,31 @@
+import { v4 as uuidv4 } from 'uuid';
+import { docClient } from '../awsDocClient.js';
+
+const putItem = async (params) => {
+  console.log('putItem params: ', params);
+  await docClient.put(params).promise();
+};
+
+export const createProduct = async (event) => {
+  const params = {
+    TableName: process.env.ProductsTableName,
+    Item: {
+      'id': uuidv4(),
+      'title': 'newItem'
```

Comment:
Why not the data from event?

Reply:
wanted to keep things looking simple (since there are no strict requirements) -- but thanks for emphasising this -- checked